src/makefile

```diff
@@ -20,13 +20,22 @@ ifeq ($(shell echo "test"), "test")
 	CFLAGS += -static
 endif
 
+# Detecting Apple Silicon (ARM64)
+UNAME := $(shell uname -m)
+ifeq ($(UNAME), arm64)
+    ARCH = arm64
+    CC = clang

```

Comment:
Should we be forcing a specific CC like this?

Reply:
No. I guess that is a left-over from being tired using 'make pgo CC=clang' :)